widgetastic_manageiq/__init__.py

```diff
@@ -5017,17 +5017,20 @@ class MigrationProgressBar(Widget):
 
     ITEM_LOCATOR = './/div[contains(@class,"card-pf-match-height")]'
     TITLE_LOCATOR = './/div[h3[contains(@class,"card-pf-title")]]'
-    TIMER_LOCATOR = './/div[contains(@class,"active-migration-elapsed-time")]'
+    TIMER_LOCATOR = './div/div[contains(@class,"active-migration-elapsed-time")]'
     SIZE_LOCATOR = './/strong[contains(@id,"size-migrated")]'
     VMS_LOCATOR = './/strong[contains(@id,"vms-migrated")]'
     SPINNER_LOCATOR = './/div[contains(@class,"spinner")]'
     ERROR_LOCATOR = './/div/span[contains(@class,"pficon-error-circle-o")]'
     PROGRESS_BARS = './/div[@class="progress-bar"]'
     PROGRESS_DESCRIPTION = './/div[contains(@class,"progress-description")]'
+    locator = './/div[contains(@class,"migrations")]/div/div/div/div/div'
```

Comment:
This is an odd locator that appears fragile to me. Is the root locator for the progress bar really buried under all those div's? Is there no class/attribute on the 5th div down that would identify it?

Reply:
BTW this is what would become `ROOT`

Reply:
No , The 5th div is something like this 
div class ="col-lg-4 col-md-6 col-sm-12"